ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.spec.ts

```diff
@@ -1,25 +1,182 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Subject } from 'rxjs';
 
 import { CronQuartzPartComponent } from './cron-quartz-part.component';
+import { WorkflowEntryModel } from '../../../../../models/workflowEntry.model';
+import { userFriendly } from '../../../../../constants/cronExpressionOptions.constants';
 
 describe('CronQuartzPartComponent', () => {
-  // let component: CronQuartzPartComponent;
-  // let fixture: ComponentFixture<CronQuartzPartComponent>;
-  //
-  // beforeEach(async(() => {
-  //   TestBed.configureTestingModule({
-  //     declarations: [ CronQuartzPartComponent ]
-  //   })
-  //   .compileComponents();
-  // }));
-  //
-  // beforeEach(() => {
-  //   fixture = TestBed.createComponent(CronQuartzPartComponent);
-  //   component = fixture.componentInstance;
-  //   fixture.detectChanges();
-  // });
-  //
-  // it('should create', () => {
-  //   expect(component).toBeTruthy();
-  // });
+  let fixture: ComponentFixture<CronQuartzPartComponent>;
```

Comment:
Tests.. that's obvious

Reply:
Ok... Please create an issue for the next sprint.

Reply:
Fixed!